addons/wp/CfgWaypointTypes.hpp

```diff
@@ -7,37 +7,36 @@ class ZEN_WaypointTypes {
     class EGVAR(danger,Garrison) {
         displayName = "1 TASK GARRISON";
         type = "SCRIPTED";
-        script = QPATHTOF(functions\fnc_taskGarrison.sqf);
+        script = QPATHTOF(scripts\fnc_wpGarrison.sqf);
     };
     class EGVAR(danger,Patrol) {
         displayName = "2 TASK PATROL";
         type = "SCRIPTED";
-        script = QPATHTOF(functions\fnc_taskPatrol.sqf);
+        script = QPATHTOF(scripts\fnc_wpPatrol.sqf);
     };
     class EGVAR(danger,Rush) {
         displayName = "3 TASK RUSH";
         type = "SCRIPTED";
-        script = QPATHTOF(functions\fnc_taskRush.sqf);
+        script = QPATHTOF(scripts\fnc_wpRush.sqf);
     };
     class EGVAR(danger,Hunt) {
         displayName = "4 TASK HUNT";
         type = "SCRIPTED";
-        script = QPATHTOF(functions\fnc_taskHunt.sqf);
+        script = QPATHTOF(scripts\fnc_wpHunt.sqf);
     };
     class EGVAR(danger,Creep) {
         displayName = "5 TASK CREEP";
         type = "SCRIPTED";
-        script = QPATHTOF(functions\fnc_taskCreep.sqf);
+        script = QPATHTOF(scripts\fnc_wpCreep.sqf);
     };
     class EGVAR(danger,CQB) {
         displayName = "6 TASK CQB [Experimental]";
         type = "SCRIPTED";
-        script = QPATHTOF(functions\fnc_taskCQB.sqf);
+        script = QPATHTOF(scripts\fnc_wpCQB.sqf);
```

Comment:
This file is currently missing and would lead to an error.

Reply:
Also do you really want to move these to scripts? having them compiled as functions would be better, so mission makers can just use them in scripts.

Reply:
1. Missing files may be due to user error on my part. Perhaps I didn't mark everything when pushing on SmartGit. (Not smart). Unfortunate.  I'll have that sorted when I get off work. 

2. In regards to the fnc_wpCQB.sqf.  The _fnc_ moniker may be wrongly chosen.  Strictly speaking I do not see any utility in keeping these as functions. Their only purpose is to exist within waypoints-- which are essentially execVMs as I understand it. 

If scripters or module makers need access to the tactical functions, it would be better to call these directly as functions. The _waypoint functions_ exist only as staging point to call or spawn the functions as appropriate function.